Exercise_B/index.html

```diff
@@ -19,16 +19,35 @@
 
   <body>
     <!--Header (flower shop logo block element)-->
+    <img src="./assets/images/logo.png" alt="logo" />
     <!--Banner Jumbotron of the image-->
-
+    <div class="hero">
+      <div class="container">
+        <h2 class="text_1">Beautiful Flowers</h2>
+        <button class="simple_button" type="button">Learn more</button>

```

Comment:
For the button to place it in the center, you can make use of 
`margin:0 auto;`